README.md

```diff
@@ -61,6 +61,63 @@ config :kafka_consumer,
 
 * Start your app.
 
+### No Consumer Group support
+
+If your Kafka setup does not support does not support consumer groups, then it is possible to still use KafkaConsumer. To still use KafkaConsumer you must
+
+* Set `offset_server` to a module that `use`'s `KafkaConsumer.OffsetServer`
+
+```elixir
+# config.exs
+
+config :kafka_ex,
+  brokers: [{"localhost", 9092}],
+  consumer_group: :no_consumer_group,
+  sync_timeout: 3000,
+  max_restarts: 10,
+  max_seconds: 60
+
+
+config :kafka_consumer,
+  default_pool_size: 5,
+  default_pool_max_overflow: 10,
+  offset_server: YourModule.OffsetServer
+  event_handlers: [
+    {KafkaConsumer.TestEventHandler, [{"topic", 0}, {"topic2", 0}], size: 5, max_overflow: 5}
+  ]
+
+# offset_server.ex
+
+defmodule YourModule.OffsetServer do
+  use KafkaConsumer.OffsetServer
+end
+```
+
+KafkaConsumer will then query this server for the current offset when appropriate.
+
+#### Starting the OffsetServer
+
+The OffsetServer is a `GenServer` and **must be started by you, and supervised by you**.
+
+It is started by calling `start_link/1`. You must provide the topic when starting the server, as the server will be named after your topic
+
+```elixir
+#starts a GenServer with a name of :that_one_topic
+YouModule.OffsetServer.start_link("that.one.topic")
+```
+
+#### Incrementing, Decrementing and Getting the offset
+
+Upon startup, KafkaConsumer will get get the **latest offset** from `KafkaEx`. After that, you must increment or decrement the offset yourself. This can be done through the `increment/1` and `decrement/1` functions. They both take the topic as the first parameter.
+
+```elixir
+YourModule.OffsetServer.increment("that.one.topic") # :ok
+YourModule.OffsetServer.decrement("that.one.topic") # :ok
+YourModule.OffsetServer.get("that.one.topic") # offset
+```
```

Comment:
I think you should also support partition number because each partition has its own offset sequence.